c2corg_ui/static/js/utils.js

```diff
@@ -26,14 +26,17 @@ app.utils.getDoctype = function(type) {
 
 
 /**
- * @param {string} document_type The document type.
+ * @param {string} documentType The document type.
  * @param {string|number} documentId The document id.
  * @param {string} lang Lang.
  * @return {string} Url.
  */
-app.utils.buildDocumentUrl = function(document_type, documentId, lang) {
-  return '/{document_type}/{id}/{lang}'
-    .replace('{document_type}', document_type)
+app.utils.buildDocumentUrl = function(documentType, documentId, lang) {
+  if (!documentType || !documentId || !lang) {
+    return '';
```

Comment:
Doing so implies changing the "return" type of the function + the type of all vars that are set with this function. That probably makes sense but this PR already does that many changes, I would rather avoid adding one more. To do in another PR?